apps.yaml

```
@@ -1,32 +1,13 @@
-aiidalab-qe:
-  categories:
-  - technology-aiida
-  - technology-aiidalab
-  - quantum
-  git_url: https://github.com/aiidalab/aiidalab-qe.git
+my-big-map-app:
```

```suggestion
dft-vasp:
```
This should be a meaningful identifier.